src/core/providers/canvas-schema/canvas-schema-vlatest.model.ts

```diff
@@ -9,6 +9,7 @@ export interface TableVm {
 }
 
 export interface FieldVm {
+  // TODO -> Add isRequired (NN) field
```

Comment:
Maybe it could be added as

```ts
isRequired?: boolean
```

And map `undefined` `null` `false`as  `false`?

Reply:
I have seen that the field name is `isNN`

Reply:
I am not sure what do you mean with the map, but I will fill `required` field from Mongo Schema only with true values of `isNN